app/views/wikis/new.html.haml

```diff
@@ -9,5 +9,12 @@
       .form-group
         = f.label :body
         = f.text_area :body, class: "form-control"
+
+      - if current_user.role == 'admin' || current_user.role == 'premium'
```

Comment:
No need to hardcode role name strings on line 13, consider using the convenience methods you added to `User`.